plugins/modules/mysql_user.py

```diff
@@ -456,8 +456,13 @@ def main():
                                             priv, append_privs, subtract_privs, tls_requires, module)
                 else:
                     changed, msg = user_mod(cursor, user, host, host_all, None, encrypted,
+<<<<<<< HEAD
                                             plugin, plugin_hash_string, plugin_auth_string,
                                             priv, append_privs, subtract_privs, tls_requires, module)
+=======
+                                            None, None, None,
+                                            priv, append_privs, tls_requires, module)
+>>>>>>> mysql_user: prevent password getting set for existing users on on_create when plugin is used
```

Comment:
```suggestion
                                            None, None, None,
                                            priv, append_privs, subtract_privs, tls_requires, module)
```
You need to fix the conflict here, i.e. add `subtract_privs` to arguments in your version

Reply:
Oh, missed that, fixed.